social_navigation/InstanceImage[] instances
int32 width
int32 height
int32 seq